app/assets/javascripts/account/AccountService.js.coffee

```diff
@@ -234,14 +237,14 @@ AccountService = (
 
   Service.userDataForSalesforce = ->
     contact = Service.userDataForContact()
-    contact.DOB = ShortFormDataService.formatUserDOB(contact)
-    contact = ShortFormDataService.removeDOBFields(contact)
-    contact
+    contactWithDOB = _.merge({}, contact, {'DOB': ShortFormDataService.formatUserDOB(contact)})
+    ShortFormDataService.removeDOBFields(contactWithDOB)
 
   Service._createAccountParams = ->
```

Comment:
@akegan while we're in here could we remove the underscore from in front of this function name? so just `createAccountParams` instead of `_createAccountParams`